pkg/cloud/openstack/machine/actuator.go

```diff
@@ -72,7 +70,7 @@ type OpenstackClient struct {
 	machineSetupWatcher *machinesetup.ConfigWatch
 	machineService      *clients.InstanceService
 	sshCred             *SshCreds
-	*DeploymentClient
+	*openstack.DeploymentClient
 }
 
 func NewMachineActuator(machineClient client.Client, scheme *runtime.Scheme) (*OpenstackClient, error) {
```

Comment:
Just nitpicking… should call it just NewActuator, to be in sync with clusteractuator.NewActuator?